zoo/auditing/admin.py

```diff
@@ -2,9 +2,8 @@
 
 from . import models
 
+admin.register(models.Issue)
+
 
```

Comment:
🤔 Hmm... shouldn't this also be using the decorator?

```suggestion
@admin.register(models.Issue)
```

Reply:
oh well... GitHub seems to ignore the empty lines on the suggestions... 😅

Reply:
Ups, yep, it seems I missed that one